tests/measureSources.py

```diff
@@ -294,6 +302,31 @@ def testPixelFlags(self):
             source,
             exp,
         )
+        # Test that if there is no center and centroider that the object should look at the footprint
+        plugin, cat = makePluginAndCat(measBase.PixelFlagsAlgorithm, "test", control)
```

Comment:
I disagree with splitting out these tests into separate tests. Within this file, each test represents testing the functionality of an individual plugin. It would be weird to have each plugin have one test, except pixel flags which has 3. And I don't really see what it gains you. Separate tests are to indicate which portion of code fails (or preferably passes). In this case that would indicate the problem is in pixel flags. To gain more specific information, one can simply look at the specific failure within the test.